solidity/random-beacon/contracts/libraries/Heartbeat.sol

```diff
@@ -56,24 +56,26 @@ library Heartbeat {
     /// @notice Verifies the failure claim according to rules mentioned in
     ///         `FailureClaim` struct documentation. Reverts if verification
     ///         fails.
+    /// @dev Group members hash is validated upstream in
+    ///      RandomBeacon.notifyFailedHeartbeat()
     /// @param sortitionPool Sortition pool used by the application performing
     ///        claim verification.
     /// @param claim Failure claim.
     /// @param group Group raising the claim.
     /// @param nonce Current nonce for group used in the claim.
+    /// @param members Group member ids that participated in dkg (excluding IA/DQ).
```

Comment:
Same comment to the documentation of this param as in `RandomBeacon.sol`.

Reply:
3589a32